Test ScriptProfileTest.filteredTimeSeriesTest failing systematically (org.openhab.core.automation.module.script)

When I build the core framework locally on my PC Windows 10, I have systematically this error:
```
[INFO] --- maven-surefire-plugin:3.2.5:test (default-test) @ org.openhab.core.automation.module.script ---
[INFO] Using auto detected provider org.apache.maven.surefire.junitplatform.JUnitPlatformProvider
[INFO]
[INFO] -------------------------------------------------------
[INFO]  T E S T S
[INFO] -------------------------------------------------------
[INFO] Running org.openhab.core.automation.module.script.internal.ScriptEngineManagerImplTest
[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.940 s -- in org.openhab.core.automation.module.script.internal.ScriptEngineManagerImplTest
[INFO] Running org.openhab.core.automation.module.script.profile.ScriptProfileTest
[main] WARN  o.o.c.a.m.s.profile.ScriptProfile - 'toHandlerScript' has been deprecated! Please use 'commandFromItemScript' instead in link 'null'.
[main] ERROR o.o.c.a.m.s.profile.ScriptProfile - Neither 'toItemScript', 'commandFromItemScript' nor 'stateFromItemScript' defined in link 'DummyItem -> foo:bar:baz:qux'. Profile will discard all states and commands.
[main] WARN  o.o.c.a.m.s.profile.ScriptProfile - 'toHandlerScript' has been deprecated! Please use 'commandFromItemScript' instead in link 'DummyItem -> foo:bar:baz:qux'.
[main] ERROR o.o.c.a.m.s.profile.ScriptProfile - Failed to process script 'inScript': intentional failure
[main] ERROR o.o.c.a.m.s.profile.ScriptProfile - Failed to process script 'inScript': intentional failure
[main] ERROR o.o.c.a.m.s.profile.ScriptProfile - Failed to process script 'outScript': intentional failure
[main] ERROR o.o.c.a.m.s.profile.ScriptProfile - Failed to process script 'outScript': intentional failure
[main] ERROR o.o.c.a.m.s.profile.ScriptProfile - Failed to process script 'inScript': intentional failure
[ERROR] Tests run: 11, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 0.153 s <<< FAILURE! -- in org.openhab.core.automation.module.script.profile.ScriptProfileTest
[ERROR] org.openhab.core.automation.module.script.profile.ScriptProfileTest.filteredTimeSeriesTest -- Time elapsed: 0.015 s <<< FAILURE!
org.mockito.exceptions.verification.TooFewActualInvocations:

transformationServiceMock.transform(
    <any>,
    <any>
);
Wanted 3 times:
-> at org.openhab.core.automation.module.script.profile.ScriptProfileTest.filteredTimeSeriesTest(ScriptProfileTest.java:348)
But was 1 time:
-> at org.openhab.core.automation.module.script.profile.ScriptProfile.executeScript(ScriptProfile.java:185)

        at org.openhab.core.automation.module.script.profile.ScriptProfileTest.filteredTimeSeriesTest(ScriptProfileTest.java:348)
        at java.base/java.lang.reflect.Method.invoke(Method.java:568)
        at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
        at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)

[INFO] Running org.openhab.core.automation.module.script.ScriptTransformationServiceTest
[main] WARN  o.o.c.a.m.s.ScriptTransformationService - Parameter 'invalid' does not consist of two parts for configuration UID scriptUid.customDsl, skipping.
[INFO] Tests run: 10, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.131 s -- in org.openhab.core.automation.module.script.ScriptTransformationServiceTest
[INFO] Running org.openhab.core.automation.module.script.TimerImplTest
[INFO] Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 27.07 s -- in org.openhab.core.automation.module.script.TimerImplTest
[INFO]
[INFO] Results:
[INFO]
[ERROR] Failures:
[ERROR]   ScriptProfileTest.filteredTimeSeriesTest:348
transformationServiceMock.transform(
    <any>,
    <any>
);
Wanted 3 times:
-> at org.openhab.core.automation.module.script.profile.ScriptProfileTest.filteredTimeSeriesTest(ScriptProfileTest.java:348)
But was 1 time:
-> at org.openhab.core.automation.module.script.profile.ScriptProfile.executeScript(ScriptProfile.java:185)

[INFO]
[ERROR] Tests run: 26, Failures: 1, Errors: 0, Skipped: 0
[INFO]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  42.701 s
[INFO] Finished at: 2024-09-28T12:30:02+02:00
[INFO] ------------------------------------------------------------------------
````

--- a/bundles/org.openhab.core.automation.module.script/src/test/java/org/openhab/core/automation/module/script/profile/ScriptProfileTest.java
+++ b/bundles/org.openhab.core.automation.module.script/src/test/java/org/openhab/core/automation/module/script/profile/ScriptProfileTest.java
@@ -358,8 +358,10 @@ public class ScriptProfileTest extends JavaTest {
 
     private TimeSeries createTimeSeries(State... states) {
         TimeSeries timeSeries = new TimeSeries(TimeSeries.Policy.ADD);
+        Instant instant = Instant.now();
         for (State state : states) {
-            timeSeries.add(Instant.now(), state);
+            timeSeries.add(instant, state);
+            instant = instant.plusMillis(100);
         }
         return timeSeries;
     }

